README.MD

```diff
@@ -10,3 +10,8 @@ cd path/to/proto/files
 
 protoc --go-pulsar_out=. --go-pulsar_opt=paths=source_relative --go-pulsar_opt=features=marshal+unmarshal+size -I .
 NAME_OF_FILE.proto
+
+
+## Acknowledgements
+
+[planetscale/vtprotobuf](https://github.com/planetscale/vtprotobuf)
```

Comment:
can we say a bit more about why were acknowledging them and that we're using code?

Reply:
hows https://github.com/cosmos/cosmos-proto/pull/42/commits/0ba1e02582b91977731053e8cffbdbc9342de885 ?

Reply:
Would be even better if we could also link to the PR/issue where that code was copied in?